Utilities/CICD/Azure-DevOps/README.md

```diff
@@ -0,0 +1,44 @@
+## CICD ADO sample
+This sample demonstrates how to integrate your app with Test Base via ADO (Azure DevOps) CI/CD.
```

Comment:
This sample demonstrates how to integrate Test Base into your CI/CD process using [Test Base REST API](https://docs.microsoft.com/en-us/rest/api/testbase/) and Azure DevOps [PowerShell Task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/powershell?view=azure-devops).